templates/che-operator/crds/org_v1_che_cr.yaml

```diff
@@ -0,0 +1,124 @@
+
```

Comment:
I didn't see any installation of the cluster roles / cluster role bindings (required for Openshift OAuth) in the typescript code.

Do we still plan to let the user add them manually to enable the Openshift OAuth case ?